index.html

```diff
@@ -618,6 +618,93 @@ <h4>
         </section>
       </section>
     </section>
+    <section id="automation" class="informative">
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document define [[WEBDRIVER2]]
+        <a href="https://w3c.github.io/webdriver/#dfn-extension-commands">
+        extension command</a> that allow controlling device posture like in real
```

Comment:
```suggestion
        extension command</a> that allows users to control the reported device posture and simulate a real
```

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/0a1ada52a79333974a0e29c1ec2eae2d52213d9a